src/plugins/data_source_management/public/components/create_data_source_wizard/components/create_form/create_data_source_form.tsx

```diff
@@ -3,76 +3,59 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import React, { Fragment } from 'react';
+import React from 'react';
 import {
   EuiButton,
   EuiFieldPassword,
   EuiFieldText,
-  EuiFlexGroup,
-  EuiFlexItem,
   EuiForm,
   EuiFormRow,
   EuiHorizontalRule,
   EuiPageContent,
-  EuiSelectable,
   EuiSpacer,
   EuiSuperSelect,
   EuiText,
 } from '@elastic/eui';
 import { FormattedMessage } from '@osd/i18n/react';
-import {
-  CreateDataSourceFormType,
-  CreateNewCredentialType,
-  CredentialsComboBoxItem,
-  CredentialSourceType,
-  DataSourceManagementContextValue,
-  ToastMessageItem,
-} from '../../../../types';
+import { credentialSourceOptions, DataSourceManagementContextValue } from '../../../../types';
 import { Header } from '../header';
-import { getExistingCredentials } from '../../../utils';
 import { context as contextType } from '../../../../../../opensearch_dashboards_react/public';
 import {
   CreateEditDataSourceValidation,
   defaultValidation,
   performDataSourceFormValidation,
-} from '../../../validation/datasource_form_validation';
+} from '../../../validation';
+import { AuthType, DataSourceAttributes, UsernamePasswordTypedContent } from '../../../../types';
+import {
+  createDataSourceCredentialSource,
```

Comment:
nit, not sure what's the convention in this repo for constants naming. e.g CREATE_DATA_SOURCE
?

Reply:
I looked at the best practices, It does not say anything .. I prefer UPPER_CASE .. I will make these changes while writing UTs